### ISSUE TITLE
$options['validation_groups'] returns string instead of array 

### ISSUE THREAD

**Issue by webdevilopers** [craue/CraueFormFlowBundle#189]
I recently updated a two month old project with `Symfony` **2.6.11** and `CraueFormBundle` **2.1.8**.

Suddenly I get this error:
**Error: [] operator not supported for strings**

This is cause by my `getFormOptions` method:
```php
    public function getFormOptions($step, array $options = array())
    {
        $options = parent::getFormOptions($step, $options);
        dump($options);
        $options['validation_groups'][] = 'priceQuoteRequest';
        return $options;
    }
```

The dump returns a string for `validation_groups`. This seemd to work fine before.
Has this behaviour changed?

Related issues:
* https://github.com/craue/CraueFormFlowBundle/issues/96
* https://github.com/craue/CraueFormFlowBundle/pull/117

**Comment by craue** [comment]
#117 has been merged into master, nothing changed in 2.1.

**Comment by webdevilopers** [comment]
Maybe the version I used before was even *1.**? Will check the `dev-master`.

**Comment by craue** [comment]
Can this be closed or do you still see an issue?

**Comment by webdevilopers** [comment]
Will be able to check it at the end of november. Just close it for now.

**Comment by webdevilopers** [comment]
I still have this issue with `2.1.8`.

**Comment by craue** [comment]
At the point where you call `dump($options)` the default value of `$options['validation_groups']` has always been a string until #117 was merged, may it be with version 1.0.0, 2.0.0, 2.1.0, or 2.1.9. Maybe you've been using dev-master initially and downgraded to 2.1.8 afterwards. Apart from that, I don't see what could explain the issue you describe. So your workaround is valid for 2.x, but wouldn't be needed anymore for upcoming 3.x.

**Comment by webdevilopers** [comment]
Ok ,closing this for now. I will either try a different branch or wait for `3.x`. Thanks
